src/correlator/twopoint.jl

```diff
@@ -86,7 +86,7 @@ Compute the polarization function of free electrons at a given frequency. Relati
 - `m`: mass
```

Comment:
add documentation for the new parameter '\mu'

Reply:
documentation updated